packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+
+#### `createPeer`
+
+<CodeBlock title="Example createPeer Mutation">
+
+```graphql
+mutation CreatePeer($input: CreatePeerInput!) {
+  createPeer(input: $input) {
+    code
+    success
+    message
+    peer {
+      id
+      asset {
+        code
+        scale
+      }
+      staticIlpAddress
+      name
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+##### Example
+
+<CodeBlock title='Example JSON request'>
+
+```json
+{
+  "input": {
+	"staticIlpAddress": "g.othergreatwallet",
+	"name": "The Other Great Wallet"
+	"http": {
+  		"incoming": {"authTokens": ["mytoken"]},
+  		"outgoing": {"endpoint": "ilp.othergreatwallet.com", "authToken": "theirtoken"}
+	  },
+	"assetId": "INSERT_ASSET_ID",
+	"initialLiquidity": <optionally, an intial amount of liquity to provision. Liquidity can also be deposited via the `DepositPeerLiquidity`>
+  }
+}
+```
+
+</CodeBlock>
+
+<div class="overflow-table">
+
+| Variable                   | Description                                                                                                                 | Required |
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `assetID`                  | The ID of the asset that you and your peer will use to ultimately settle your net obligations outside of Interledger.       | Y        |
+| `staticILPaddress`         | Your peer’s static ILP address (the example uses `g.othergreatwallet`)                                                      | Y        |
+| `name`                     | Your peer’s name (the example uses “The Other Great Wallet”)                                                                | Y        |
+| `http.incoming.authTokens` | The token your peer will use to present and connect to and send packets to your Rafiki instance.                            | Y        |
+| `http.outgoing.endpoint`   | Your peer’s connector endpoint. By default it is on local port 0.0.0.0:3002                                                 | Y        |
+| `http.outgoing.authtoken`  | The token that you will use to present to your peer and connect to and send packets to their Rafiki instance.               | Y        |
+| `initialLiquidity`         | Initial amount of liquidity to deposit for peer. Liquidity can also be deposited using the `DepositPeerLiquidity` mutation. | N        |
+
+</div>
+
+<CodeBlock title="Example JSON response">
+
+```json
+{
+  "data": {
+    "createPeer": {
+      "code": "200",
+      "success": true,
+      "message": "Created ILP Peer",
+      "peer": {
+        "id": "480ef339-7842-4501-a905-923fc1339cef",
+        "asset": {
+          "code": "USD",
+          "scale": 2
+        },
+        "staticIlpAddress": "g.othergreatwallet",
+        "name": "The Other Great Wallet"
+      }
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+### Create wallet addresses
```

Comment:
Let's talk about how to handle this section. It reads as if the ASE needs to create a wallet address for their peer, and I don't believe that's right. 

Also it says "refer to Assets" and links to Wallet Addresses :)

Reply:
Removing "adding wallet address" for now.